src/Support/Facades/Input.php

```diff
@@ -18,6 +21,18 @@ public static function get($key = null, $default = null)
         return static::$app['request']->input($key, $default);
     }
 
+    /**
+     * Gets all input data items.
+     *
+     * This method is used for all request verbs (GET, POST, PUT, and DELETE)
+     *
+     * @return array|null
+     */
+    public static function all()
+    {
+        return static::$app['request']->input();
+    }
```

Comment:
Why was this required?

Reply:
Mainly for the `input()` helper method - it will try to retrieve all input if no name is provided to that method. This is synonymous with the `post()` and `get()` helper methods.